public/r2/v1/swagger/swagger.json

```diff
@@ -84,7 +84,7 @@
                         }
                     },
                     "409": {
-                        "description": "Namespace already exists",
+                        "description": "The ruleset got updated while you were looking at it.",
```

Comment:
Hm I thought we'd return 409 when the namespace already existed? If someone calls this API programmatically without checking whether the namespace exists beforehand, they can get 409 if they try to create a namespace that exists. If so, probably wanna keep it as "Namespace already exists"?